Python/compile.c

```diff
@@ -4154,6 +4154,7 @@ expr_constant(struct compiler *c, expr_ty e)
         else if (o == Py_False)
             return 0;
     }
+    /* fall through */
```

Comment:
Wouldn't be better to move this comment one line up, inside a block, and align it with `if` and `else`? And add "else" before "fall through".